pineappl_cli/tests/uncert.rs

```diff
@@ -24,56 +24,57 @@ Options:
   -h, --help                  Print help
 ";
 
-const DEFAULT_STR: &str = "b   etal    dsig/detal  PDF central    PDF    
-     []        [pb]                    [%]    
--+----+----+-----------+-----------+-----+----
-0    2 2.25 7.5459110e2 7.5461655e2 -1.14 1.14
-1 2.25  2.5 6.9028342e2 6.9027941e2 -1.16 1.16
-2  2.5 2.75 6.0025198e2 6.0022595e2 -1.18 1.18
-3 2.75    3 4.8552235e2 4.8548211e2 -1.22 1.22
-4    3 3.25 3.6195456e2 3.6191001e2 -1.27 1.27
-5 3.25  3.5 2.4586691e2 2.4582640e2 -1.35 1.35
-6  3.5    4 1.1586851e2 1.1584074e2 -1.51 1.51
-7    4  4.5 2.7517266e1 2.7504644e1 -2.77 2.77
+const DEFAULT_STR: &str =
+    "b   etal    dsig/detal    NNPDF31_nlo_as_0118_luxqed       NNPDF40_nnlo_as_01180   
+     []        [pb]                                                                
```

Comment:
wouldn't it be better to have ` [] [pb] [pb] [%] [%] [pb] [%] [%]` ? such that it is more plain that the 6th column is a _relative_ uncert? (here, by chance, it is almost clear) (also just one `[%]` as before would work)

Reply:
Fixed in commit a981f7e2241410c1c73bc495aebdda3c7a64e306.

The problem is that the [`prettytable`](https://crates.io/crates/prettytable-rs) crate supports only a single title row. We therefore use a linebreak to simulate a second row, but in this particular case we use a single title for three columns, and together with the linebreak this effectively disables the correct centering of the units. However, I agree that even with the wrong formatting having the units is important.